### PR TITLE
issue-3-edit-an-entry | Angel Leung

### PR DESCRIPTION
**Pull request by angelsleung**
issue-3-edit-an-entry

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -71,10 +71,12 @@ textarea {
   padding: 0.3rem 0.6rem;
   border-radius: 4px;
   border: none;
+  cursor: pointer;
 }
 
-.button:hover {
+.fas.fa-edit {
```
Comment (codingcodymiller) [inline css/styles.css]: You don't ever want to add styling to classes from other libraries. If another developer on your team wanted to use this icon, they might wonder why it suddenly has a pointer cursor. Rather, apply this styling to a custom class, and add that class to your edit icon element

File: js/main.js [hunk 2]
```diff
@@ -75,6 +103,29 @@ function viewEntries(event) {
...
+  inputPhotoUrl.value = entryObject.photoUrl;
+  placeholderImage.setAttribute('src', entryObject.photoUrl);
+  inputNotes.value = entryObject.notes;
+}
+
+function getEntryObject(entryListElement) {
+  var entryId = entryListElement.getAttribute('data-entry-id');
+  var numEntries = data.entries.length;
+  var entryIndex = numEntries - entryId;
+  var entryObject = data.entries[entryIndex];
```
Comment (codingcodymiller) [inline js/main.js]: You should not depend on an `id` to be the same as its index like this. For instance, what if eventually the item with the id `4` gets deleted? Now this won't work. Instead, you should loop over your list of entries and find the entry with the matching id.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Good work so far!! Please see my notes

File: css/styles.css (hunk 1 above)
Comment (angelsleung) [reply css/styles.css]: Yeah makes sense! Thank you!

File: js/main.js (hunk 2 above)
Comment (angelsleung) [reply js/main.js]: Ahh yes I forgot about the "Delete an entry" feature of this project 😅 Thank you for the helpful pointers!

**Review by codingcodymiller — APPROVED** [review]
🥳 🥳 🥳

**Pull request merged**